stdlib/public/core/StringCharacterView.swift

```diff
@@ -156,10 +156,9 @@ extension String.CharacterView : BidirectionalCollection {
   ///     // Prints "[72, 101, 97, 114, 116, 115]"
   public struct Index : Comparable, CustomPlaygroundQuickLookable {
     public // SPI(Foundation)    
-    init(_base: String.UnicodeScalarView.Index) {
+    init(_base: String.UnicodeScalarView.Index, in c: String.CharacterView) {
```

Comment:
I see, it is not really possible to preserve the old SPI.  OK, probably we'd need a coordinated pull request for Foundation then!